vhdbuilder/packer/components.json

```diff
@@ -190,25 +190,18 @@
         "v3.23.1"
       ]
     },
-    {
-      "downloadURL": "mcr.microsoft.com/oss/cilium/operator:*",
-      "amd64OnlyVersions": [],
-      "multiArchVersions": [
-        "1.12.1"
-      ]
-    },
     {
       "downloadURL": "mcr.microsoft.com/oss/cilium/operator-generic:*",
       "amd64OnlyVersions": [],
       "multiArchVersions": [
-        "1.12.1"
+        "1.12.2"
       ]
     },
     {
       "downloadURL": "mcr.microsoft.com/oss/cilium/cilium:*",
       "amd64OnlyVersions": [],
       "multiArchVersions": [
-        "1.12.1.1"
+        "1.12.2"
```

Comment:
no 4th version tuple? just sanity checking

Reply:
nope, just x.y.z (the last part is appended by upstream publishing when they release a security patch)